app/src/main/java/com/hamdan/forzenbook/data/network/NetworkResources.kt

```diff
@@ -2,6 +2,12 @@ package com.hamdan.forzenbook.data.network
 
 object NetworkResources {
     const val EMAIL = "email"
-    const val LOGIN = "login"
-    const val CREATE_USER = "createUser/"
+    const val CODE = "code"
+    const val BIRTH_DATE = "birth_date"
+    const val FIRST_NAME = "first_name"
+    const val LAST_NAME = "last_name"
+    const val LOCATION = "location"
+    const val LOGIN_GET = "login"
+    const val LOGIN_POST = "login/"
```

Comment:
These Path variables should be in the service, these aren't ever going to be used elsewhere

Reply:
ok moved corresponding constants to corresponding services.